frontend/simple/views/Invite.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <section class="section full-screen">
+    <div class="columns">
+      <div class="column is-1"></div>
+        <div class="column is-10" >
+          <div class="columns is-gapless">
+            <div class="column">
+              <p class="control has-addons">
+                <input id="searchUser" class="input" type="text" v-model="searchUser" placeholder="Find a member">
+                <a id="addButton" class="button is-info" v-on:click="add">
+                  <i18n>Add</i18n>
+                </a>
+              </p>
+              <span v-if="error" id="badUsername" class="help is-danger"><i18n>Invalid Username</i18n></span>
+            </div>
+            <div class="column">
+              <table class="table is-bordered is-striped is-narrow">
+                <thead>
+                <tr>
+                  <th><i18n>Group Members</i18n></th>
+                </tr>
+                </thead>
+                <tbody>
+                <tr v-for="(member, index) in members" class="member">
+                  <td>
+                    <div class="media">
+                      <div class="media-left">
+                        <p class="image is-64x64">
+                          <img src="http://bulma.io/images/placeholders/128x128.png">
+                        </p>
+                      </div>
+                      <div class="media-content">
+                        <strong>{{member.name}}</strong>
+                      </div>
+                      <div class="media-right">
+                        <button class="delete" v-on:click="remove(index)"></button>
+                      </div>
+                    </div>
+                  </td>
+                </tr>
+                </tbody>
+              </table>
+              <div class="center button-box">
+                <div class="center">
+                  <div id="successMsg" v-if="invited" class="created"><i18n>Success</i18n></div>
+                  <button class="button is-success is-large center" v-if="!invited" :disabled="!members.length" v-on:click="submit" type="submit"><i18n>Invite Members</i18n></button>
+                  <a class="button is-warning is-large center" v-if="invited"><i18n>Next: ?</i18n></a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+       <div class="column is-1"></div>
+    </div>
+  </section>
+</template>
+<script>
+import fetch from 'node-fetch'
+import request from 'superagent'
+import * as Events from '../../../shared/events'
+import _ from 'lodash'
```

Comment:
FYI `mapValues` is one of the lodash functions I implemented in a couple of lines in `js/utils`, so it's better to use that version if that's the only function you're going to use, since at some point we might simply get rid of lodash as a dependency (to save on file size).